### DISCUSSION TITLE
Hub Rename

### DISCUSSION THREAD

**Discussion by robbysibrahim** [MeetHub-io/meethub-support#7] [Q&A]
How do we rename a hub? I created one called Office Printer thinking that I was creating rooms but instead it seems to be the global account name.

**Comment by maxmil** [comment]
This hasn't been implemented yet. Although I can see why it would be useful.

In the meantime if you tell me the name you're like I can change it for you.

**Comment by robbysibrahim** [comment]
We can rename it to "Pluto TV" for us please. Thanks!

**Reply by maxmil** [reply]
Done
